demo/staging.vals.yaml

```diff
@@ -7,6 +7,10 @@ server:
   image:
     tag: edge
     pullPolicy: Always
+    credentials:
+      registry: hub.instrumentisto.com
```

Comment:
@logist322 please, do not make private repos public anywhere. Specify them in local `my.*.vals.yaml` files only, which are ignored by Git.